src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,41 @@
+.header {
+  background-image: url(../images/header.jpg);
+  background-position: center;
+  background-size: cover;
+  padding: 25px 20px;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+    max-width: 1020px;
+    margin: 0 auto;
+    margin-bottom: 0;
```

Comment:
Change paddings, not margins, if you wanna control inner content of block, also it'd be better to remove 80px margin from your h1
```suggestion
.header {
  background-image: url(../images/header.jpg);
  background-position: center;
  background-size: cover;
 
  &__container {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    height: 100vh;
    max-width: 1020px;
    margin: 0 auto; 
    padding: 25px 20px 80px;

```